src/closure.js

```diff
@@ -31,28 +36,50 @@ const limitFunctionCallCount = (cb, n) => {
   };
   // Should return a function that invokes `cb`.
   // The returned function should only allow `cb` to be invoked `n` times.
+  let stuff=0;
+  return (...args) => {
+     if (stuff < n) {
+       stuff++;
```

Comment:
You should use a more specific variable name here.  `stuff` is too generic and doesn't describe what the variable is holding